packages/lab/src/scrim/Scrim.css

```diff
@@ -8,7 +8,7 @@
   height: 100%;
   display: flex;
   z-index: 1199;
```

Comment:
One to check with design probably, but should this be `--salt-zIndex-drawer` or a new var on foundations?

Reply:
yeah maybe a new var

Reply:
For now you should use calc(var(--salt-zIndex-drawer) - 1)